inference-engine/src/preprocessing/ie_preprocess_gapi_kernels.cpp

```diff
@@ -2520,9 +2600,10 @@ GAPI_FLUID_KERNEL(FDivC, GDivC, false) {
 using namespace kernels;
 
 cv::gapi::GKernelPackage preprocKernels() {
-    return cv::gapi::kernels
-        < FChanToPlane
-        , FScalePlanes
+    return combine(
+        FColorConversionChooseISA(),
+        cv::gapi::kernels
+        <FScalePlanes
```

Comment:
I believe this static list will reduce with the time as you move all other kernels to this new method?

Reply:
Yes, it will.